### PR TITLE
SIDM-3769 - Update idam-perftest

### PR DESCRIPTION
**Pull request by henrydobson**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/SIDM-3769

### Change description ###

Update idam-perftest

NOTE: Due to the tagging approach, this also requires a deletion of the existing deployments to ensure that the deployment is updated correctly.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: k8s/perftest/common/idam/idam-web-public.yaml [hunk 1]
```diff
@@ -14,11 +14,12 @@ spec:
   chart:
     repository: https://hmctspublic.azurecr.io/helm/v1/repo/
     name: idam-web-public
-    version: 0.2.2
+    version: 0.2.3
   values:
     java:
       image: hmctspublic.azurecr.io/idam/web-public:perttest
```
Comment (dfourn) [inline k8s/perftest/common/idam/idam-web-public.yaml]: this says perttest and not perftest, is that correct?

File: k8s/perftest/common/idam/idam-web-public.yaml [hunk 2]
```diff
@@ -14,11 +14,12 @@ spec:
...
-      ingressHost: idam-web-public.service.core-compute-perftest.internal
+      imagePullPolicy: Always
+      ingressHost: idam-web-public.perftest.platform.hmcts.net
```
Comment (dfourn) [inline k8s/perftest/common/idam/idam-web-public.yaml]: Is that the correct url to use here, should it not be the internal one?

File: k8s/perftest/common/idam/idam-web-public.yaml [hunk 3]
```diff
@@ -14,11 +14,12 @@ spec:
...
       aadIdentityName: idam
```
Comment (kremi) [inline k8s/perftest/common/idam/idam-web-public.yaml]: Shall we add an envvar overwrite for
```
STRATEGIC_SERVICE_URL: http://idam-api-perftest.service.core-compute-perftest.internal
```

File: k8s/perftest/common/idam/idam-web-public.yaml (hunk 1 above)
Comment (henrydobson) [inline k8s/perftest/common/idam/idam-web-public.yaml]:
```suggestion
      image: hmctspublic.azurecr.io/idam/web-public:perftest
```

**Pull request merged**